packages/nimble-components/src/table-column/base/index.ts

```diff
@@ -50,6 +57,22 @@ export abstract class TableColumn<
         this.columnInternals = new ColumnInternals(options);
         this.columnInternals.currentSortDirection = this.sortDirection;
         this.columnInternals.currentSortIndex = this.sortIndex;
+        this.validator = {
+            isValid(): boolean {
+                return true;
+            },
+            getValidity(): ValidityObject {
+                return {};
+            }
+        };
```

Comment:
We should have an actual ColumnValidator class that can be extended. We generally have not used the pattern shown here (and frankly thought there was a lint run against it)

Reply:
Used this approach.

Reply:
I had originally misunderstood the approach to take, but have since fixed it. The only issue is that it is slightly awkward when deriving from `ColumnValidator`, because you have to pass the array of validation flag names in to the type parameter  of `ColumnValidator`. See `TestColumnValidator` in `table-column.fixtures.ts`.